src/components/JobConfiguration.vue

```diff
@@ -125,7 +125,8 @@ export default defineComponent({
       getJobStatus: 'job/getJobStatus',
       getJob: 'job/getJob',
       shopifyConfigId: 'user/getShopifyConfigId',
-      currentEComStore: 'user/getCurrentEComStore'
+      currentEComStore: 'user/getCurrentEComStore',
+      getCurrentEComStore:'user/getCurrentEComStore',
```

Comment:
I think new getter is not required, we could use `currentEComStore`

Reply:
Removed this unwanted getter.